posts/2024-04-20-Performance Comparison: Data.table in R and Pandas in Python-Doris Amoakohene/index.qmd

```diff
@@ -159,7 +162,7 @@ atime.read <- atime::atime(
 refs.read.vary.cols <- atime::references_best(atime.read)
 plot(refs.read.vary.cols)
 ```
-
+The graph provides a closer look at larger values of N, indicating that data.table::fread outperforms pandas::read_csv, particularly as N increases. complexity notations such as log N, N log N, and N^2 are used to describe the performance characteristics.
```

Comment:
capitalize C in complexity because it starts the sentence.
performance characteristics -> asymptotic time/memory usage

Reply:
updated